docs/references/cli.md

```diff
@@ -1,26 +0,0 @@
-# Command reference
-
```

Comment:
Why are we removing guidance on how to build good CLI content?

Reply:
@samccann , ok, i put it back.